packages/scripts/utils/test/index.js

```diff
@@ -76,6 +99,80 @@ describe( 'utils', () => {
 		} );
 	} );
 
+	describe( 'getJestOverrideConfigFile', () => {
+		beforeEach( () => {
+			getArgsFromCLIMock.mockReturnValue( [] );
```

Comment:
It looks like even more mocking was required than in other tests but I applaud your efforts. Tests read good so I guess it's a great thing to have since the logic is quite complex 👍 

It also raises the flag for me, I should improve the test coverage when working on the code in the past 😅

Reply:
Yeah, unfortunately the function uses quite an assortment of the other utility functions. I tried to keep the mocking as high-level as possible to try to limit how deep I had to go with it. Not sure there's much better that can be done here. It might be good in general if the entire test file could use more "common" mocks, and limit the specific mocking here to the various scenarios that would affect the implementation.